README.md

```diff
@@ -219,9 +219,8 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
-[OPTIONAL] kindly consider removing this text as this is referred to the writer of this readme, not the reader or visitor. removing this kind of text will make your readme more professional.